minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Mixins/MixinRandomSkinTexture.java

```diff
@@ -0,0 +1,27 @@
+
+
+package com.microsoft.Malmo.Mixins;
+
+
+import com.microsoft.Malmo.Utils.SeedHelper;
+import net.minecraft.client.resources.DefaultPlayerSkin;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.Random;
+import java.util.UUID;
+
+
+@Mixin(DefaultPlayerSkin.class)
```

Comment:
nice!

Reply:
I wonder if we should make this optional/controllable from python. For example, it could be useful to give each team in a multi-agent setting a different skin.

Reply:
I think that is a good idea - likely to be included in the team based PR